definitions/npm/backbone_v1.x.x/flow_v0.42.x-/backbone_v1.x.x.js

```diff
@@ -0,0 +1,322 @@
+// flow-typed signature: ba506076507fa389f4f828349e657080
```

Comment:
Should be removed.

Reply:
Good point, I made a PR to prevent this from happening altogether: https://github.com/flow-typed/flow-typed/pull/2835